src/utils/crypto.utils.ts

```diff
@@ -0,0 +1,10 @@
+import {uint8ArrayToHexString} from '@dfinity/utils';
+
+export async function generateHash(params: Record<string, string>): Promise<string> {
+  const sortedKeys = Object.keys(params).sort();
+  const jsonString = JSON.stringify(params, sortedKeys);
```

Comment:
- `sortedKeys` are second paremeter of `JSON.stringify` seems incorrect. Is this expected?

- To follow-up my question about generic object, if we do so, I would then suggest here to use the `jsonReplacer` from `@dfinity/utils`.